scripts/enforce-yaml-quotes.ts

```diff
@@ -0,0 +1,37 @@
+import * as fs from "fs";
```

Comment:
This script is run during the lint-fence-spell step. Its main purpose is to enforce consistent formatting by converting all strings in our CMS collection YAML files to use double quotes. This helps prevent formatting drift and ensures clean diffs when parsing/writing YAML programmatically.

We’re using js-yaml with forceQuotes: true and quotingType: '"' to make sure everything in web/decap-config/collections/ is quote-consistent across the board.